tools/travis-test.sh

```diff
@@ -54,13 +60,6 @@ setup_base()
     build_args+=("--simd-test=\$werror BASELINE SSE2 SSE42 XOP FMA4 (FMA3 AVX2) AVX512F AVX512_SKX VSX VSX2 VSX3 NEON ASIMD VX VXE VXE2")
   fi
   if [ -z "$USE_DEBUG" ]; then
-    # activates '-Werror=undef' when DEBUG isn't enabled since _cffi_backend'
-    # extension breaks the build due to the following error:
-    #
-    # error: "HAVE_FFI_PREP_CIF_VAR" is not defined, evaluates to 0 [-Werror=undef]
-    # #if !HAVE_FFI_PREP_CIF_VAR && defined(__arm64__) && defined(__APPLE__)
-    #
-    export CFLAGS="$CFLAGS -Werror=undef"
     $PYTHON setup.py build "${build_args[@]}" install 2>&1 | tee log
   else
```

Comment:
Still need setup here?

Reply:
Yes, because the TravisCI jobs are for testing SIMD features, so we should only port them over once the Meson SIMD support lands (that is much closer now, but not there yet).